cli/src/program.rs

```diff
@@ -2982,6 +3022,37 @@ fn report_ephemeral_mnemonic(words: usize, mnemonic: bip39::Mnemonic) {
     eprintln!("[BUFFER_ACCOUNT_ADDRESS] argument to `solana program close`.\n{divider}");
 }
 
+fn fetch_feature_set(
```

Comment:
A lot of people have asked about adding the ability for `solana-test-validator` to mimic the feature set on a network, so I might end up reusing this very useful function, thanks!